frontend/assessments/1/style.scss

```diff
@@ -0,0 +1,107 @@
+
+    body {
+        background-color: pink; 
+        font-family: 'Poppins', sans-serif;
+        color: white;
+    }
+    nav {
+        background-color: pink;
+        overflow: hidden;
+        position: fixed;
+        width: 100%; 
+        top: 0; 
+        z-index: 1000;
+        .heading{
+            display: inline;
+        text-align: left;
+        height: 40px;  
+        }
+        .hamburger{
+            display: none;
+        }
+        ul{
+            display: inline;
+            list-style-type: none;
+            margin: 0;
+            padding: 500px;
+            overflow: hidden;
+            li{
+                float: right;
+                a{
+                    display: inline ;
+                    color: white;
+                    text-align: center;
+                    padding: 10px 16px;
+                    text-decoration: none;
+                }
+                a:hover{
+                    background-color: #ddd;
```

Comment:
Add all the hard-coded values in the different file as constants and then import them